.github/CODEOWNERS

```
@@ -39,7 +39,7 @@
 /src/pytorch_lightning/core/module.py        @williamfalcon @tchaton @awaelchli @carmocca
 
 # Lightning Fabric
-/src/lightning_fabric                         @awaelchli @carmocca @justusschock
+/src/lightning/fabric                         @awaelchli @carmocca @justusschock
```

```suggestion
/src/lightning_fabric                         @awaelchli @carmocca @justusschock
```
skip and do it at once :chipmunk: